tools/report-converter/report_converter/ReportConverter.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python
```

Comment:
With moving it out from here the converter directory could be removed and every converter could be moved under `report_converter` and not `report_converter/converter`.

On the other maybe it would be better to name the package `codechecker_report_converter` the same naming strategy we use in `codechecker_analyzer`, and `codechecker_web` so when it is installed in a python environment it is quite easily visible and probably will not conflict.